jst-nuke-init

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env node
+
+var exec = require('child_process').execSync
+var inquirer = require('inquirer')
+var clone = require('gitclone')
+var rimraf = require('rimraf')
+var replaceInFile = require('replace-in-file')
+var cd = process.chdir
+var GitHubApi = require('github')
+
+var github = new GitHubApi()
+
+var replace = function (replace, withWhat) {
+  replaceInFile.sync({
+    files: ['package.json', 'README.md', 'index.js'],
+    replace: replace,
+    with: withWhat
+  })
+}
+
+var errHandler = function (err) {
+  if (err) { console.error(err.stack) }
+}
+
+var print = function (data) {
+  console.log(data.stdout)
+  console.error(data.stderr)
+}
+
+var run = function (command) {
+  console.log(exec(command).toString())
+}
+
+var getInfo = function () {
+  return inquirer.prompt([
+    {
+      name: 'humanName',
+      message: 'Name of JSTransformer (capitalized properly)'
+    },
+    {
+      name: 'url',
+      message: 'URL to find more about the transformer'
+    },
+    {
+      name: 'username',
+      message: 'GitHub Username (for creating repo)'
+    },
+    {
+      name: 'password',
+      type: 'password',
+      message: 'GitHub Password (for creating repo)'
+    }
+  ]).then(function (answers) {
+    github.authenticate({
+      type: "basic",
+      username: answers.username,
+      password: answers.password
+    }, function () {console.log('authenticated with GitHub')})
```

Comment:
This is not going to work with 2FA (which I use). Check https://github.com/mikedeboer/node-github#creating-a-token-for-your-application for how to create a token and use it here.

Also, please use promises.